posts/faraycloudx/Invest-on-testing-tool.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Why you should invest on a testing plugin or complements for your project' 
+published: false
+tags: 'qa, management, testing, testingtools, optimization'
+cover_image: ./assets/testing-tool/coverTestingTool.webp
+---
+
+## Overview
+
+A common topic in the software development process is the budget for testing tools. Nowadays it is still common to see multiple excel sheets for test cases, emails with bug reports or documents with regressions suites reports. Because of this the market offers a large number of testing tools, but not every tool can be right for you and is sometimes part of our job to explain why the client or the team should invest part of the budget on a testing plugin or a testing management tool. In the article below I will try to explain some of the main reasons to include these powerful complements on your project.
+
+## How much does a testing plugin or complements cost?
+
+There are multiple management tools on the market like Jira, Azure Devops that are used around the world but the testing plugins are not all free and sometimes the free options do not meet the needs of the team. Let's see some prices of the most common plugins available on the market:
+
+### For Jira  
+
+**Zephyr Squad or Scale**: Up to 10 users. USD 229. per year.  
+**Tricentis qTest**: starts at USD $1,000 per user, per year.  
+**Xray**: Up to 10 users. USD 10. per year.  
+**Ones Test case**: USD $7.99 per user, per month 1-100 users.  
+**Testrail Professional Cloud**: USD $408 per user, per year.  
+**Testrail Professional Server**: USD $8140 Up to 10 users, per year.  
+
+## What should you consider to make a choice?
+
+FFirst you need to know what types of testing are you going to do and make sure the plugin or complements you want to buy have the set that enables you to perform all the tasks. Consider that this tool is going to make your work easier but also needs to be useful for your partners on your team. For example: reports, real-time test results, management of test cases, test executions, regressions suites, option to create and store scripts, integration with automated testing frameworks, continuous integration are some of the main features you need to consider. Second, consider also the documentation or support that the tool has, because it's important to also have good support just in case you will have issues on installation or use. Third, make sure that the tool should help you and your team with the development work. The tool should adapt to your processes, not the opposite.
```

Comment:
```suggestion
First you need to know what types of testing are you going to do and make sure the plugin or complements you want to buy have the set that enables you to perform all the tasks. Consider that this tool is going to make your work easier but also needs to be useful for your partners on your team. For example: reports, real-time test results, management of test cases, test executions, regressions suites, option to create and store scripts, integration with automated testing frameworks, continuous integration are some of the main features you need to consider. Second, consider also the documentation or support that the tool has, because it's important to also have good support just in case you will have issues on installation or use. Third, make sure that the tool should help you and your team with the development work. The tool should adapt to your processes, not the opposite.
```